.github/workflows/tests.yaml

```diff
@@ -25,6 +25,34 @@ jobs:
           cd api
           DJANGO_SECRET_KEY="testkey" python3 manage.py test
 
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Run ShellCheck
+        run: |
+          shellcheck service/bin/* snap/hooks/*
```

Comment:
Can you enclose there paths in double quotes?

Reply:
`*` does not expand inside quotes